assets/less/icon.less

```diff
@@ -1,5 +1,5 @@
 @theme-version: '1.10.1';
-@font-base-path: "//unpkg.com/jsonresume-theme-elegant@@{theme-version}/assets/icomoon/fonts";
+@font-base-path: './assets/icomoon/fonts';
```

Comment:
The issue with this approach is that, as far as I know when we publish a resume, it will only accept the `index.js`, so the assets won't be served from the theme server. Which is why I had to go via the Icomoon approach. Because that way, I could import the font files via a hosted URL rather than theme server serving it.